src/transformers/utils/hub.py

```diff
@@ -49,7 +52,9 @@
     RevisionNotFoundError,
     build_hf_headers,
     hf_raise_for_status,
+    send_telemetry,
 )
+from huggingface_hub.utils._deprecation import _deprecate_method
```

Comment:
Should we make this a public method to leverage?

Not ideal to depend on private attributes from hfh 🫣

Reply:
Yeah I realized about that while working on this PR :sob: 
I'm fine with either making them public (`huggingface_hub.utils.deprecate_method` ?) which would require to wait next hfh release or making an exception for this one since it's a quite internal method anyway (so using private attribute in transformers).

Reply:
As mentioned in https://github.com/huggingface/transformers/pull/27564#issuecomment-1819097559 I suggest to still use the private attribute until it's made more official in hfh side. Worse case scenario, the plan is to remove this in the next releases.